app/models/user.rb

```diff
@@ -46,14 +46,17 @@ class User < ApplicationRecord
   scope :locked, (-> { where(locked: true) })
   scope :unlocked, (-> { where(locked: false) })
 
-  scope :admins, (-> { where(admin: true) })
+  scope :admins, (-> { where(role: :admin) })
+  scope :conf_admins, (-> { where(role: :conf_admin) })
```

Comment:
this scope seams not to be used anywhere ... remove it if so !

Reply:
Done